setLocation.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"github.com/go-telegram-bot-api/telegram-bot-api"
+	"encoding/json"
+    "net/http"
+    "net/url"
+	"strconv"
+)
+
+type CepResponse struct {
+                            CEP        string  `json:"cep"`
+                            Logradouro string  `json:"logradouro"`
+                            Bairro     string  `json:"bairro"`
+                            Cidade     string  `json:"cidade"`
+                            Estado     string  `json:"estado"`
+                            Latitude   string  `json:"latitude"`
+                            Longitude  string  `json:"longitude"`
+                            Altitude   float64 `json:"altitude"`
+                            DDD        int     `json:"ddd"`
+                            IBGE       string  `json:"ibge"`
+                        }
+
+// locationHandler receive postal code from user.
+func locationHandler(update tgbotapi.Update, bot *tgbotapi.BotAPI, config botConfig, msgs botMessages) error {
+
+	isCep := false
+	pais := "??"
+	cep := ""
+	user := update.Message.From
+
+	if update.Message.Command() == "setlocation" {
+		args := strings.Split(strings.Trim(update.Message.CommandArguments(), " "), " ")
```

Comment:
You use update.Message.CommandArguments() a number of times. Consider saving it in a variable (say, "args")